packages/reports/addon/components/filter-values/dimension-select.ts

```diff
@@ -58,7 +58,10 @@ export default class DimensionSelectComponent extends Component<DimensionSelectC
   get selectedDimensions() {
```

Comment:
Can we explicitly type this
```suggestion
  get selectedDimensions(): NaviDimensionModel[] {
```